### PR TITLE
Corner button story

### PR DESCRIPTION
**Pull request by dorechen**

### CODE REVIEW

File: src/CornerButton.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+const baseButton = css`
+  border: none;
+  height: 48px;
+  min-width: 204px;
+  background: #000;
+  color: #fff;
+  font-size: 24px;
+  cursor: pointer;
+  transition: linear 300ms;
```
Comment (gouegd) [inline src/CornerButton.js]: let's specify what needs to transition: `linear background 300ms`

File: src/CornerButton.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  @media print {
+    display: none;
+  }
+`
+
+const mobileButton = css`
+  min-width: 48px;
+`
+
+export const CornerButton = ({ mobile, ...props }) => (
```
Comment (gouegd) [inline src/CornerButton.js]: do we have a defintion of a CornerButton, actually ? (as opposed to other 'normal' Buttons)

Comment (dorechen) [reply src/CornerButton.js]: Yes, a corner button will be floating at the bottom right corner. (and slight size and colour difference)
Though I wonder if it's better to not include the positioning styles in the button component, in cases where we want more than one button on the corner (e.g. the "Kiosk" and "Delete" button on this screen). 
image

A possible solution: we could have an invisible wrapping component with absolute positioning at the bottom of the parent, and right content alignment.

What do you think?

Comment (dorechen) [reply src/CornerButton.js]: We also use corner buttons in the "pop-up modules"
image

Comment (gouegd) [reply src/CornerButton.js]: Interesting, thanks.
I agree with having a wrapping component. Maybe something like "ActionArea", i.e.
`<ActionArea><CornerButton ... /><ActionArea>` ?

**Pull request merged**